onnxscript/analysis_test.py

```diff
@@ -163,5 +163,57 @@ def nested():  # pylint: disable=unused-variable
         self.assertUses(f, {"x"})
 
 
+class TestAssignedVarAnalysis(unittest.TestCase):
+    def assertDefs(self, f, expected: set[str]):
+        source, parse_tree = ast_utils.get_src_and_ast(f)
+        result = analysis.assigned_vars(parse_tree.body, formatter(source))
+        self.assertEqual(result, expected)
+
```

Comment:
I would lift this out and make sure what is being asserted is clear from its name

```suggestion
def assert_defined_variables(self, f, expected: set[str]):
    source, parse_tree = ast_utils.get_src_and_ast(f)
    result = analysis.assigned_vars(parse_tree.body, formatter(source))
    self.assertEqual(result, expected)


class TestAssignedVarAnalysis(unittest.TestCase):

```

Reply:
Renamed it. But leaving it inside the class ... that seems preferable to me.

Reply:
ok